src/rust/ide/view/graph-editor/src/component/visualization/container/action_bar.rs

```diff
@@ -0,0 +1,289 @@
+//! Definition of the QuickActionBar component.
+
+use crate::prelude::*;
+
+use crate::component::node;
+use crate::component::visualization::container::visualization_chooser;
+use crate::component::visualization;
+
+
+use enso_frp as frp;
+use enso_frp;
+use ensogl::application::Application;
+use ensogl::data::color;
+use ensogl::display::shape::*;
+use ensogl::display::traits::*;
+use ensogl::display;
+use ensogl::gui::component;
+use ensogl_text as text;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+const HOVER_COLOR    : color::Rgba = color::Rgba::new(1.0,0.0,0.0,0.000_001);
```

Comment:
Please use theme manager for that, and show how it looks in light mode.

Reply:
This is not a real visible, color. Just an almost transparent overlay to get mouse events.